Sources/RDeviceIdentifier/RDeviceIdentifier.swift

```diff
@@ -0,0 +1,179 @@
+import UIKit.UIDevice
+import CommonCrypto
+#if !canImport(RSDKUtils)
+import RLogger // Required in SPM version
+#endif
+
+@objc public final class RDeviceIdentifier: NSObject {
```

Comment:
I think I'd vote for removing objc support - any reason to keep it? @malayli

We would also need to remove the wrapper class from Analytics but we need to do that anyway for SPM.

Reply:
We can remove `@objc`. @apparition47 Can you remove @objc and NSObject inheritance?